docs/installation/binary_installation.md

```diff
@@ -0,0 +1,58 @@
+# Binary Installation
+
+Visualization packages with CARET can also be installed in binary.
+
+<prettier-ignore-start>
+!!! Note
+    Only the visualization packages can be installed by binary installation.
+    The binary does not include measurement packages.
+<prettier-ignore-end>
```

Comment:
It is better to describe exactly what `Visualization packages` is.

Reply:
Fixed in https://github.com/tier4/caret_doc/pull/335/commits/6d8b79ba4e972a18a2bd51d9b9b28008d4180576.

Reply:
Sorry, I think the sentence I pointed out was a bit vague and the content was not good.

In many cases, users do not understand the exact relationship between a package and the functionality it provides.
Perhaps a new user does not know which features are available in the documentation if they install from this page.

Reply:
added url for visualization and recording. User can confirm what function can use. Fixed in https://github.com/tier4/caret_doc/pull/335/commits/75c09bd65aacace23ca985435bcb8b9a75c31aef